### PR TITLE
feat: UI แบบ 3000%

### PR DESCRIPTION
**Pull request by tinarskii**
image

### CODE REVIEW

**Comment by Leomotors** [comment]
wtf is that commit message

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -45,34 +44,48 @@ importers:
       vitest: 0.24.3
 
 packages:
-
   /@ampproject/remapping/2.2.0:
-    resolution: {integrity: sha512-qRmjj8nj9qmLTQXXmaR1cck3UXSRMPrbsLJAasZpF+t3riI71BXed5ebIOYwQntykeZuhjsdweEc9BxH5Jc26w==}
-    engines: {node: '>=6.0.0'}
+    resolution:
```
Comment (Leomotors) [inline pnpm-lock.yaml:48]: bro, did you just run format on pnpm lock file?

Comment (tinarskii) [reply pnpm-lock.yaml:48]: no prettierignore file?

**Comment by tinarskii** [comment]
> wtf is that commit message

squash and merge and no more cursed

**Review by Leomotors — APPROVED** [review]
lgtm no review, i trust you, if prod down open another pr

File: pnpm-lock.yaml (hunk 1 above)
Comment (Leomotors) [reply pnpm-lock.yaml]: we really need that?

Comment (Leomotors) [reply pnpm-lock.yaml]: ok

**Pull request merged**